pkg/gardener/shoot/converter.go

```diff
@@ -98,7 +100,8 @@ func NewConverterPatch(opts PatchOpts) Converter {
 			opts.MachineImage.DefaultVersion,
 			opts.Workers,
 			opts.InfrastructureConfig,
-			opts.ControlPlaneConfig))
+			opts.ControlPlaneConfig,
```

Comment:
If I'm not mistaken, then the logger will still be a `nil` without additionally adding that to PatchOpts - here's a PR to your PR - https://github.com/akgalwas/infrastructure-manager/pull/3